judoscale-rails/lib/judoscale/rails/metrics_collector.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "judoscale/web_metrics_collector"
+require "judoscale/metric"
+require "judoscale/config"
+require "judoscale/rails/utilization_middleware"
+
+module Judoscale
+  module Rails
+    class MetricsCollector < Judoscale::WebMetricsCollector
+      def collect
+        metrics = super
+
+        if Judoscale::Config.instance.utilization_enabled
+          # Report utilization percentage as a whole number (floats not supported)
+          idle_ratio = UtilizationTracker.instance.idle_ratio
+          utilization = (1.0 - idle_ratio) * 100.0
+          metrics.push(Metric.new(:up, utilization, Time.now)) # Utilization percentage
```

Comment:
Ah wait, now I see, the plan is to send this as a different metric then... At first I thought we were replacing the original one. I misunderstood that part.

Right now we send multiple metrics with 0/1 values and aggregate on our side, this will be a single metric per/report. Gotcha. 🤔